src/main/java/com/rarchives/ripme/ui/MainWindow.java

```diff
@@ -208,14 +212,9 @@ private void statusWithColor(String text, Color color) {
 
     private void pack() {
         SwingUtilities.invokeLater(() -> {
-            boolean collapsed = isCollapsed();
-            if (!collapsed) {
-                mainFrame.setResizable(true);
-            }
             Dimension preferredSize = mainFrame.getPreferredSize();
             mainFrame.setMinimumSize(preferredSize);
-            if (collapsed) {
-                mainFrame.setResizable(false);
+            if (isCollapsed()) {
```

Comment:
This if block snaps the window size back to the regular packed size - if you don't like this, we can remove it

Reply:
I think it's fine